neo/Ledger/BlockChain.State.cs

```diff
@@ -13,7 +13,7 @@ namespace Neo.Ledger
     public sealed partial class Blockchain : UntypedActor
     {
         public class ImportRoots { public IEnumerable<StateRoot> Roots; }
-        public uint StateHeight => currentSnapshot.StateHeight;
+        public long StateHeight => currentSnapshot.StateHeight == uint.MaxValue ? -1 : (long)currentSnapshot.StateHeight;
```

Comment:
uint.MaxValue as flag?
```suggestion
        public uint StateHeight => currentSnapshot.StateHeight;
```

if it's max value -> sync from the beginning.

Reply:
It's the default value of `HashIndexState`. Any suggestion?

Reply:
`StateHeight` could be `From` inclusive, if it's 0, it will sync from 0 inclusive?

Reply:
Yes. Block 1 will confirm state of genesis block.
If we enable at 0, `StateHeight` isn't 0 before block 1 generated.